packages/remote-config/src/api2.ts

```diff
@@ -39,3 +43,26 @@ export async function fetchAndActivate(
   await fetchConfig(remoteConfig);
   return activate(remoteConfig);
 }
+
+/**
```

Comment:
@eglimorez This is the other documentation block, it should be identical to the text in the compat/index.d.ts file.